data.sql

```diff
@@ -68,4 +68,125 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody P
 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
-WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon', 'Boarmon');
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17');
+       ('Stephanie Mendez', 64, '1981-05-04');
+       ('Jack Harkness', 38, '2008-06-08');
```

Comment:
You have done excellent work so far and I like the way you make the code DRY by using one ```INSERT INTO``` statement instead of writing a new one for each value 🔥 👍. However, I suggest that you remove the semi-colons at the end of each value except the last value so as to as the current state throws errors. The code might look something like this in the end

```sh
INSERT INTO vets (name, age, date_of_graduation)
VALUES ('William Tatcher', 45, '2000-04-23'),
       ('Maisy Smith', 26, '2019-01-17'),
       ('Stephanie Mendez', 64, '1981-05-04'),
       ('Jack Harkness', 38, '2008-06-08');
```